src/blog/2024/02/node-red-perfect-adapter-middleware-uns.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Node-RED: The perfect adapter and middleware for your UNS"
+subtitle: The broker is the data backbone for the unified namespace, which one is right for you?
+description: The broker is the data backbone for the unified namespace, which one is right for you?
+date: 2024-02-05
+authors: ["zeger-jan-van-de-weg", "marian-demme"]
+image: # TODO /blog/2024/01/images/broker-for-uns.png
+tags:
+    - posts
+    - flowfuse
+    - unified-namespace
+---
+
+Digitalization is at the inflection point where it’s been adopted enough that the additional investments provided better and better ROIs for organizations. The next bump in ROI will be achieved through the UNS. A torrent of information is more useful when structured and adapted for new use-cases. Either a [performance dashboard](/blueprints/manufacturing/performance-overview/), Artificial Intelligence, or station metrics – each is built faster when the data is readily available and well structured? As a company started around Node-RED, we’ve not spoken a lot where FlowFuse fits into the picture, which is what this post is about.
```

Comment:
It's a good idea to link to the blueprint.

Reply:
Thank you